io/disk/multipath_test.py

```diff
@@ -185,9 +185,41 @@ def test(self):
             for path in path_dic['paths']:
                 if multipath.reinstate_path(path) is False:
                     msg += "%s did not recovered  in all path fail\n" % path
+
+            # Removing all paths for short time and adding back
+            self.log.info("Removing all paths and adding back")
+            for path in path_dic['paths']:
+                if multipath.remove_path(path) is False:
```

Comment:
@Naresh-ibm the return is a boolean and you can easily do `if not multipath.remove_path(path):` instead of `is false`: